pyteal/__init__.py

```diff
@@ -1,69 +1,69 @@
 from pyteal.ast import *
 from pyteal.ast import __all__ as ast_all
-from pyteal.pragma import pragma
```

Comment:
Originally I needed to touch this file in order to "load()" the `FeatureGates` from the `.ini` if there was one. In touching the file, I re-sorted the imports.  I can revert to the original if I get a request.